Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.mapbox.vision.examples;
+
+import androidx.annotation.NonNull;
+
+import com.mapbox.vision.ar.FenceVisualParams;
+import com.mapbox.vision.ar.LaneVisualParams;
+import com.mapbox.vision.ar.VisionArManager;
+import com.mapbox.vision.ar.core.models.Color;
+import com.mapbox.vision.ar.view.gl.VisionArView;
+
+public class ArCustomizationActivity extends ArActivity {
+
+    @Override
+    protected void setArRenderOptions(@NonNull final VisionArView visionArView) {
+        // Make sure lane is visible
+        visionArView.setLaneVisible(true);
+        // Set the desired lane length in meters.
+        VisionArManager.setLaneLength(40f);
+
+        // Create an instance of LaneVisualParams
```

Comment:
I'd remove this comment - it doesn't describe our API

Reply:
OK